src/components/vm/overview/firmware.tsx

```diff
@@ -50,7 +64,7 @@ class FirmwareModal extends React.Component {
         this.save = this.save.bind(this);
     }
 
-    dialogErrorSet(text, detail) {
+    dialogErrorSet(text: string, detail: string) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2185-af23a7d6-20250716-112918-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)